sdk/metricsadvisor/azure-ai-metricsadvisor/samples/async_samples/sample_authentication_async.py

```diff
@@ -54,6 +54,28 @@ async def authentication_administration_client_with_metrics_advisor_credential_a
     # [END administration_client_with_metrics_advisor_credential_async]
 
 
+async def authentication_client_with_aad_async():
+    # [START authentication_client_with_aad_async]
```

Comment:
Can we add a comment here about setting the env vars for DefaultAzureCredential? Something like: 

"DefaultAzureCredential will use the values from these environment variables: AZURE_CLIENT_ID, AZURE_TENANT_ID, AZURE_CLIENT_SECRET"

Reply:
DefaultAzureCredential is a chain of credentials and does not require AZURE_CLIENT_ID, AZURE_TENANT_ID, AZURE_CLIENT_SECRET to work. e.g. if users have managed identity configured, they don't need to configure anything to make DefaultAzureCredential work.